contracts/sources/obligation.move

```diff
@@ -366,9 +461,16 @@ module suilend::obligation {
 
         assert!(is_healthy(obligation), EObligationIsNotHealthy);
 
-        if (isolated(config(reserve)) || obligation.borrowing_isolated_asset) {
+        if (isolated(config(reserve)) || obligation.borrowing_isolated_asset || is_emode) {
```

Comment:
nit: eh, lets just explicitly check num_borrows == 1 in the is_emode branch below

Reply:
done